networkit/cpp/graph/test/GraphToolsGTest.cpp

```diff
@@ -222,6 +222,40 @@ TEST_P(GraphToolsGTest, testRandomNode) {
     }
 }
 
+TEST_P(GraphToolsGTest, testRandomNodes) {
+    constexpr count n = 20;
+    constexpr double p = 0.01;
+
```

Comment:
Here `constexpr` is not needed, we do not need to evaluate `n` and `p` at compile time. In these cases I think it's fine to just use `const`, for more check here https://en.cppreference.com/w/cpp/language/constexpr